dids/src/main/kotlin/web5/sdk/dids/methods/key/DidKey.kt

```diff
@@ -55,10 +56,10 @@ public class CreateDidKeyOptions(
  * ### Usage Example:
  * ```kotlin
  * val keyManager = InMemoryKeyManager()
- * val did = DidKey("did:key:example", keyManager)
+ * val did = DidKey.load("did:key:example", keyManager)
```

Comment:
Should this usage example be moved above ln 119?

Reply:
Done